test/liblll/Parser.cpp

```diff
@@ -174,6 +174,12 @@ BOOST_AUTO_TEST_CASE(list)
 	BOOST_CHECK(!successParse("()"));
 }
 
+BOOST_AUTO_TEST_CASE(macro_with_zero_args)
+{
+	char const* text = "(def 'panic () (asm INVALID))";
```

Comment:
Can you change it here too for clarity? (You are right, change is not needed here.)